tensorflow/python/kernel_tests/cwise_ops_test.py

```diff
@@ -1772,6 +1772,14 @@ def testSqrt(self):
 
 class RoundingTest(tf.test.TestCase):
 
+  def _compare_values(self, x, y=None, use_gpu=True):
```

Comment:
What do you mean by force test?  If the GPU is available and the kernel is registered, it will be tested on GPU.  Otherwise it will be on CPU.